client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-location-list.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import List from 'woocommerce/components/list/list';
+import ListItem from 'woocommerce/components/list/list-item';
+import ListHeader from 'woocommerce/components/list/list-header';
+import ListItemField from 'woocommerce/components/list/list-item-field';
+import { bindActionCreatorsWithSiteId } from 'woocommerce/lib/redux-utils';
+import { getCurrentlyEditingShippingZoneCountries } from 'woocommerce/state/ui/shipping/zones/locations/selectors';
+
+const ShippingZoneLocationList = ( { loaded, translate, onChange } ) => {
+	const renderLoaction = ( method, index ) => {
+		if ( ! loaded ) {
+			return (
+				<ListItem key={ index } className="shipping-zone__location is-placeholder" >
+					<ListItemField className="shipping-zone__location-title">
+						<span />
+					</ListItemField>
+					<ListItemField className="shipping-zone__location-summary">
+						<span />
+						<span />
+					</ListItemField>
+					<ListItemField className="shipping-zone__location-actions">
+						<Button compact >{ translate( 'Edit' ) }</Button>
+					</ListItemField>
+				</ListItem>
+			);
+		}
+
+		return (
+			<ListItem key={ index } className="shipping-zone__location" >
+				<ListItemField className="shipping-zone__location-title">
+				</ListItemField>
+				<ListItemField className="shipping-zone__location-summary">
+				</ListItemField>
+				<ListItemField className="shipping-zone__location-actions">
+					<Button compact >{ translate( 'Edit' ) }</Button>
+				</ListItemField>
+			</ListItem>
+		);
+	};
+
+	const onAddLocation = () => {
+		if ( ! loaded ) {
+			return;
+		}
+		onChange();
+	};
+
+	const locationsToRender = [ {}, {}, {} ];
+
+	return (
+		<div className="shipping-zone__locations-container">
+			<ExtendedHeader
+				label={ translate( 'Zone locations' ) }
+				description={ translate( 'Add locations that you want to share shipping methods' ) } >
+				<Button onClick={ onAddLocation } disabled={ ! loaded } >{ translate( 'Add location' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **46** times:
`translate( 'Add Location' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).